.circleci/.reviewdog.yml

```diff
@@ -0,0 +1,15 @@
+# reviewdog.yml
+
+runner:
+  golint-fetcher:
```

Comment:
Go ahead and take out all of the `-fetcher` portions of the command names in this file.